collections.mdx

```diff
@@ -16,7 +16,45 @@ You can view our example app
 [Polybase Social](https://social.testnet.polybase.xyz) to see it working in
 action.
 
-## Create a collection
+## Create a collection programmatically
+
+If you want to create your collection programatically, see [Creating collection with applySchema](/collections#).
```

Comment:
This should be in the get-started section, and the link should link to the title `## Create a collection programmatically`